### DISCUSSION TITLE
[#1040903] Editing Mob/Block NBT

### DISCUSSION THREAD

**Discussion by JWForever5504** [Mojang/minecraft-editor#16] [Feedback]
In Bedrock Edition, we require external tools to access NBT data of many blocks/entities. What if editor mode allowed us to edit mobs' and blocks' NBT data? What could be the uses?

**Comment by minecraftchloe** [comment]
We have plans to expose and edit this info neatly in the Editor. Stay tuned, it's something we definitely want to improve the workflow on within the context of the Editor.

**Reply by conmaster2112** [reply]
I know it not a right place to ask, but if you plan of somekind of editing NBT or just data components of entities/block/item, will be that possible to do it in script api as well or it will be like editor API only?
